ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetTransactionCount.java

```diff
@@ -45,14 +62,56 @@ public String getName() {
 
   @Override
   public JsonRpcResponse response(final JsonRpcRequest request) {
-    if (request.getParamLength() != 2) {
+    if (request.getParamLength() == 2) {
```

Comment:
Don't like the idea of differentiating the method based on number of arguments, seems a bit nasty. Perhaps this should change to use a parameter object like what is done with createPrivacyGroup?